### DISCUSSION TITLE
Unable to fetch data from multiple URIs

### DISCUSSION THREAD

**Discussion by EmadGKamel** [grafana/grafana-json-datasource#171] [Q&A]
I tried to leave the "URI" field blank in the settings to be able to make API calls from multiple URIs, but I am failing to call those URIs in the dashboard.
How can I call multiple URIs from the same dashboard using this plugin?

**Comment by marcusolsson** [comment]
The hostname is currently the defining part of a data source. You'll have to create separate data sources for each hostname. You can reuse the same hostname with different paths however.
